source/fundamentals/enterprise-auth.txt

```diff
@@ -0,0 +1,216 @@
+
+.. include:: /includes/fundamentals/auth-specify.rst
+
+Mechanisms
+----------
+
+.. _gssapi-auth-mechanism:
+
+Kerberos (GSSAPI/SSPI)
+~~~~~~~~~~~~~~~~~~~~~~
+
+*Available in MongoDB Enterprise Edition 3.7 and later.*
+
+The Generic Security Services API (``GSSAPI``) authentication mechanism
+allows the user to authenticate to a Kerberos service using the user's
+principal name.
+
+.. note::
+   The method refers to the ``GSSAPI`` authentication mechanism instead
+   of ``Kerberos`` because the driver authenticates using the
+   `GSSAPI RFC-4652 <https://tools.ietf.org/html/rfc4752>`_  SASL
+   mechanism.
+
+The following code snippets show how to specify the authentication mechanism,
+using the following placeholders:
+
+* ``username`` - your URL-encoded principal name, e.g. "username%40REALM.ME"
+* ``hostname`` - network address of your MongoDB server, accessible by your client
+* ``port`` - port number of your MongoDB server
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions and sample code for specifying this authentication
+mechanism:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify the GSSAPI authentication mechanism using a connection
+      string, assign the ``authMechanism`` parameter to the value
+      ``GSSAPI`` in your connection string. Set the ``authSource`` parameter
+      to the value ``$external``. Your code to instantiate a ``MongoClient`` 
+      should resemble the following:
+
+      .. code-block:: java
+
+         MongoClient mongoClient = MongoClients.create("<username>@<hostname>:<port>/?authSource=<authenticationDb>&authMechanism=GSSAPI");
+
+   .. tab::
+      :tabid: MongoCredential
+
+      To specify the GSSAPI authentication mechanism using the
+      ``MongoCredential`` class, use the ``createGSSAPICredential()``
+      method. Your code to instantiate a ``MongoClient`` should resemble the
+      following:
+
+      .. include:: /includes/fundamentals/code-snippets/auth-credentials-gssapi.rst
+
+
+In order to acquire a Kerberos ticket, the GSSAPI Java libraries may require
+that you specify the realm and Key Distribution Center (KDC) properties. See
+the sample settings in the example below:
+
+.. code-block:: properties
+
+   java.security.krb5.realm=MYREALM.ME
+   java.security.krb5.kdc=mykdc.myrealm.me
+
+You may need to specify one or more of the following additional properties
+depending on your Kerberos setup:
+
+- ``SERVICE_NAME``
+- ``CANONICALIZE_HOST_NAME``
+- ``JAVA_SUBJECT``
+- ``JAVA_SASL_CLIENT_PROPERTIES``
+
+Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
+tab below for instructions on how to specify these properties:
+
+.. tabs::
+
+   .. tab::
+      :tabid: Connection String
+
+      To specify one of the GSSAPI additional properties, include it in the
+      connection string as a parameter using the format:
+      ``<PROPERTY_NAME>:<value>``.
```

Comment:
I'll clarify that it is a URL parameter (it might be appended with a "?" depending on whether it's appended first). I'm not sure we need to explain it further than that, but perhaps I'll add an external link here or in the Connection String page.